README.md

```diff
@@ -1,72 +1,42 @@
 # Deis Slugbuilder v2
 
 [![Build Status](https://travis-ci.org/deis/slugbuilder.svg?branch=master)](https://travis-ci.org/deis/slugbuilder)
+[![Docker Repository on Quay](https://quay.io/repository/deisci/slugbuilder/status "Docker Repository on Quay")](https://quay.io/repository/deisci/slugbuilder)
 
-Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy and manage
-applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
-a lightweight, [Heroku-inspired](http://heroku.com) workflow.
+Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any [Kubernetes](http://kubernetes.io) cluster, making it easy to deploy and manage applications on your own servers.
 
-## Work in Progress
+For more information about the Deis Workflow, please visit the main project page at https://github.com/deis/workflow.
 
-![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
+## Beta Status
 
-Deis Slugbuilder v2 is changing quickly. Your feedback and participation are more than welcome, but be
-aware that this project is considered a work in progress.
+This Deis component is currently in beta status, and we welcome your input! If you have feedback, please [submit an issue][issues]. If you'd like to participate in development, please read the "Development" section below and [submit a pull request][prs].
 
 # About
 
-Deis Slugbuilder takes a git archive and compiles a slug and puts it in endpoint which can be a S3 based object store or file server. It's launched by [`deis/builder`] containers when procfile-based application code is `git push`ed. Please see https://github.com/deis/builder/blob/master/README.md for a more detailed description of how `slugbuilder` interacts with `builder`.
+The slugbuilder downloads a git archive ([gzip](http://www.gzip.org/)ped [tar](https://www.gnu.org/software/tar/)ball) from a specified [S3 API compatible server][s3-api], compiles a [slug](https://devcenter.heroku.com/articles/slug-compiler) and uploads it to a specified S3 API compatible server.
```

Comment:
@smothiki these docs definitely should have that information in here, but I'd like to punt that to a future PR so that we can at least get a foundation in. Can you create an issue for adding that info?